company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract.md

```diff
@@ -16,13 +16,19 @@ All Mattermost Staff Members, including our CEO, who are spending money or signi
 
 ### TL;DR
 
-* \*\*\*\*[**Step 1**](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract#step-1-send-for-budget-approval): Complete the [Purchase Order Form](https://docs.google.com/forms/d/e/1FAIpQLSd0zdXhAxYcH59wDnUoprgDi3aLdMBk3K3d3nNSmsqB2jtwAw/viewform?usp=sf_link) to get spend approval. This will kick-off the order requisition process and a Purchase Order Number \(PO-\#\) will be assigned
-* \*\*\*\*[**Step 2**](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract#step-2-request-contract-from-vendor): If not an [MTA](https://handbook.mattermost.com/operations/finance/risk-management#mattermost-templated-agreement), request and send contracts to the [Procurement Team](mailto:procurement@mattermost.com)
-* \*\*\*\*[**Step 3**](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract#step-3-legal-review-and-approval): [LRA](https://handbook.mattermost.com/operations/finance/risk-management#mattermost-templated-agreement)
-* \*\*\*\*[**Step 4**](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract#step-4-send-for-internal-signature): Send document to be signed internally
-* \*\*\*\*[**Step 5**](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract#step-5-send-signed-contract-for-counter-signature): Obtain a counter-signed document
-* \*\*\*\*[**Step 6**](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract#step-6-upload-completed-contract): Send the final counter-signed agreement to the [Procurement Team](mailto:procurement@mattermost.com)
-* \*\*\*\*[**Step 7**](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract#step-7-vendor-on-boarding-and-invoice-processing): Follow the vendor on-boarding process
+[**Step 1**](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract#step-1-send-for-budget-approval): Complete the [Purchase Order Form](https://docs.google.com/forms/d/e/1FAIpQLSd0zdXhAxYcH59wDnUoprgDi3aLdMBk3K3d3nNSmsqB2jtwAw/viewform?usp=sf_link) to get spend approval. This will kick-off the order requisition process and a Purchase Order Number \(PO-\#\) will be assigned
```

Comment:
```suggestion

- [**Step 1**](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract#step-1-send-for-budget-approval): Complete the [Purchase Order Form](https://docs.google.com/forms/d/e/1FAIpQLSd0zdXhAxYcH59wDnUoprgDi3aLdMBk3K3d3nNSmsqB2jtwAw/viewform?usp=sf_link) to get spend approval. This will kick-off the order requisition process and a Purchase Order Number \(PO-\#\) will be assigned

```